include/evm_runtime/evm_contract.hpp

```diff
@@ -5,12 +5,13 @@
 #include <eosio/ignore.hpp>
 
 #include <evm_runtime/types.hpp>
+#include <evm_runtime/transaction.hpp>
+#include <evm_runtime/runtime_config.hpp>
+#include <evm_runtime/config_wrapper.hpp>
```

Comment:
We shouldn't need to bring in `tables.hpp` (which is an implementation detail) into the header file for the contract. This is now being brought in via `config_wrapper.hpp`.

We should have a `config_wrapper.hpp` and a `config_wrapper.cpp` where the cpp is the one that brings in `tables.hpp` and implements all the accessor and mutators.